qa/tasks/cephfs/test_acls.py

```diff
@@ -9,6 +9,9 @@
 
 class TestACLs(XFSTestsDev):
 
+    def setup_xfsprogs_devs(self):
+        self.install_xfsprogs = False
```

Comment:
Is this intentional? Shouldn't this be initialized to `True` since this class derives from ` XFSTestsDev` and overloads `setup_xfsprogs_devs` (otherwise, there is no use of the overload, isn't it)?

Reply:
It's intentional. We don't need to install the `xfsprogs` from source for `TestACLs`.

Since the `xfs_io` command version is not high enough to support the `add_enckey` subcommad in some OSs, such as the `Centos 8` and `RHEL 8`. And the `add_enckey` will be needed for the fscrypt test cases only.

I will improve it and define this in parent class and set it to `False` as default.